evm/vm/forks/sharding/vmc_handler.py

```diff
@@ -212,9 +305,10 @@ def send_transaction(self,
             gas_price=gas_price,
             data=data,
         )
-        build_transaction_instance = self.buildTransaction(build_transaction_detail)
-        func_instance = getattr(build_transaction_instance, func_name)
-        unsigned_transaction = func_instance(*args)
+        func_instance = getattr(self.functions, func_name)
```

Comment:
@carver we should make it possible to do `self.functions[func_name]` in web3.py contracts.

Reply:
:+1: tracking in https://github.com/ethereum/web3.py/issues/705